sorts/library_sort.py

```diff
@@ -0,0 +1,108 @@
+"""
+This is a pure Python implementation of the library sort algorithm.
+
+For doctests run following command:
+python -m doctest -v library_sort.py
+or
+python3 -m doctest -v library_sort.py
+For manual testing run:
+python library_sort.py
+"""
+
+
+def library_sort(collection: list) -> list:
+    """
+    Sorts the given collection in ascending order using the Library Sort algorithm.
+
+    :param collection: A mutable ordered collection with comparable items.
+    :return: The same collection ordered by ascending.
+
+    Examples:
+    >>> library_sort([0, 5, 3, 2, 2])
+    [0, 2, 2, 3, 5]
+    >>> library_sort([])
+    []
+    >>> library_sort([-2, -5, -45])
+    [-45, -5, -2]
+    """
+
+    def insertion_sort(arr):
```

Comment:
We already have an implementation of insertion sort in this repo. Why not import that implementation so that you can keep your implementation simpler?